lib/groupmem.c

```diff
@@ -105,7 +105,7 @@ bool gr_append_member(struct group *grp, char *member)
 	}
 
 	for (i = 0; grp->gr_mem[i]; i++) ;
```

Comment:
Huh.  Does anything use gr_append_member?  I wonder if we can drop it.

Reply:
Not that I'm aware of